searx/infopage/en/donate.md

```diff
@@ -0,0 +1,16 @@
+# Donate to SearXNG
+
+* Credit/debit card and bank transfer
+    * [Liberapay] (recurrent donation)
+    * [Buy Me a Coffee] (one time donation)
+
+* Cryptocurrency
+    * Bitcoin: bc1qn3rw8t86h05cs3grx2kmwmptw9k4kt4hyzktqj (Segwit compatible)
+    * Bitcoin cash: bitcoincash:qpead2yu482e3h9amy5zk45l8qrfhk59jcpw3cth9e
```

Comment:
Why is there a `bitcoincash:` before the address? With the new bitcoin cash address, you can't unintentionally send bitcoin to a bitcoin cash format address.